### ISSUE TITLE
variable names dont really match their use

### ISSUE THREAD

**Issue by TobyHFerguson** [TobyHFerguson/Corralitos#8]
the variable named uek-iso (or similar) is simply the boot iso - it should be renamed to make its role clearer

**Comment by UncleRedBeard** [comment]
i'll add this update to the `ol6u6_vars.json` file in lvmi9-002 branch

`uek_iso_url` renamed to `uek_boot_iso_url`

**Comment by TobyHFerguson** [comment]
Nope. Should be a separate branch an change for all versions. Think small!

**Comment by UncleRedBeard** [comment]
dangit...i jumped the gun and already added it to lvmi9-002
